library/scan_sudoers.py

```diff
@@ -0,0 +1,579 @@
+#!/usr/bin/python
```

Comment:
We will need some python unit tests for this module. Maybe the source has some tests.

Reply:
I am adding python unit tests to CI in https://github.com/linux-system-roles/sudo/pull/23